sonic_api.yaml

```diff
@@ -1255,6 +1255,122 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 #----------------------------------------------
+# VRF config API
+#----------------------------------------------
+  '/config/vrf/{vrf_id}':
+    post:
+      summary: Create a new VRF representing a virtual routing instance. If VRF exists, 409 conflict error will be returned
+      parameters:
+        - name: vrf_id
+          in: path
+          required: true
+          type: string
+          description: vrf_id can be a guid
+      responses:
+        '204':
+          description: OK
+        '400':
+          description: Malformed arguments for API call
+          schema:
+            $ref: '#/definitions/Error'
+        '401':
+          description: Invalid authentication credentials
+          schema:
+            $ref: '#/definitions/Error'
+        '409':
+          description: Resource Conflict
+          schema:
+            $ref: '#/definitions/Error'
+        '403':
+          description: Capacity insufficient
+          schema:
+            $ref: '#/definitions/Error'
+        '500':
+          description: Internal service error
+          schema:
+            $ref: '#/definitions/Error'
+        '503':
+          description: Maintanence mode
+          schema:
+            $ref: '#/definitions/Error'
+    get:
+      summary: Get information about an existing VRF
+      description: Returns attributes for requested vrf_id.
+      parameters:
+        - name: vrf_id
+          in: path
+          required: true
+          type: string
+          description: vrf_id representing the virtual routing instance
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: object
+            required:
+              - vrf_id
+              - attr
+            properties:
+              vrf_id:
+                type: string
+              attr:
+                $ref: '#/definitions/VrfEntry'
```

Comment:
VrfEntry definition is missing. Please add in this PR.

Reply:
Removed.